src/language-js/printer-estree.js

```diff
@@ -1558,6 +1558,19 @@ function printPathNoParens(path, options, print, args) {
           n.elements.length > 1 &&
           n.elements.every((element, i, elements) => {
             const elementType = element && element.type;
+
+            if (n.type === "ArrayExpression") {
+              let i = 0;
+              let node = path.getNode(i);
```

Comment:
@yogmel You shouldn't have moved this loop inside the other loop that iterates through the elements of the array literal. Please move it back.